travis/shared_configs/doctr-upload.yml

```diff
@@ -0,0 +1,39 @@
+jobs:
+  include:
+    - stage: deploy
+      name: "Docs Upload"
+      install:
+
+        - echo "Req File':' ${DOCS_REQUIREMENTS:=docs-requirements.txt}"
+        - echo "Folder':' ${DOCS_FOLDER:=docs}"
+        - |
+          # Install docs requirements
+          if [[ -f "${DOCS_REQUIREMENTS}" ]]; then
```

Comment:
This may help debugging:
```bash
if [[ ! -z "${DOCS_REQUIREMENTS}" && ! -f "${DOCS_REQUIREMENTS}" ]]; then
  exit 1
fi
```

Reply:
Implemented something similar in a recent commit.